tests/test_file_extension.py

```diff
@@ -0,0 +1,64 @@
+#!/usr/bin/python3 -OO
+# Copyright 2007-2021 The SABnzbd-Team <team@sabnzbd.org>
+#
+# This program is free software; you can redistribute it and/or
+# modify it under the terms of the GNU General Public License
+# as published by the Free Software Foundation; either version 2
+# of the License, or (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program; if not, write to the Free Software
+# Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+
+"""
+Testing SABnzbd correct extension functionality module
+"""
+
+import os
+from tests.testhelper import *
+import sabnzbd.utils.file_extension as file_extension
+
+
+class TestPuremagic:
+    def test_puremagic_magic_file(self):
+        import puremagic  # direct access only in this test function
```

Comment:
I don't see the point of this test? Since it's testing basically `puremagic` itself.
If you want to make sure `par2` works, just make a copy of this file (but strip the extension) and then run our own functions on it.